proto/v2/v2.proto

```diff
@@ -19,11 +19,22 @@ package datacommons.v2;
 
 import "entity.proto";
 
-message PropertyValuesV2Request {
+message Arc {
+  string property = 1;
+  repeated datacommons.EntityInfo nodes = 2;
+}
+
+message QueryV2Request {
+  // Nodes to query for.
   repeated string nodes = 1;
+  // The expression for the property or the chained property.
   string property = 2;
```

Comment:
Should we include a summary of the language we decided here?

Reply:
Done